subprojects/dependency-management/src/main/java/org/gradle/internal/component/external/model/AbstractMutableModuleComponentResolveMetadata.java

```diff
@@ -231,6 +233,14 @@ public void setSource(ModuleSource source) {
         this.moduleSource = source;
     }
 
+    public void setAttributes(ImmutableAttributes attributes) {
```

Comment:
`@Override` annotation missing

Reply:
Not with the current source level (`1.6`). With `1.7` it would work, but here it would be a compile error.